commit c5ee17b65e0c6bbd725049ccbb78bdd225ca7f42
Author: Thomas Genty <tomlohave@openbricks.org>
Date:   Sun Sep 6 10:01:09 2015 +0200

    imx6qdl-cm-fx6 : remove pu_dummy, use reg_3p3v instead
    Now wm8731-audio is working.

diff --git a/arch/arm/boot/dts/imx6qdl-cm-fx6.dtsi b/arch/arm/boot/dts/imx6qdl-cm-fx6.dtsi
index e8f80bc..a317c56 100644
--- a/arch/arm/boot/dts/imx6qdl-cm-fx6.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-cm-fx6.dtsi
@@ -183,9 +183,6 @@
 		status = "okay";
 	};
 
-	pu_dummy: pudummy_reg {
-		compatible = "fsl,imx6-dummy-pureg"; /* only used in ldo-bypass */
-	};
 };
 
 &iomuxc {
@@ -545,10 +542,10 @@
 		reg = <0x1a>;
 		clocks = <&clks 173>, <&clks 158>, <&clks 201>, <&clks 200>;
 		clock-names = "pll4", "imx-ssi.1", "cko", "cko2";
-		AVDD-supply = <&pu_dummy>;
-		HPVDD-supply = <&pu_dummy>;
-		DCVDD-supply = <&pu_dummy>;
-		DBVDD-supply = <&pu_dummy>;
+		AVDD-supply = <&reg_3p3v>;
+		HPVDD-supply = <&reg_3p3v>;
+		DCVDD-supply = <&reg_3p3v>;
+		DBVDD-supply = <&reg_3p3v>;
 	};
 };
 
